scania-digital-design-system/sdds
# Feature/theme structure

**Describe pull-request**  
- Move out core from light theme
- Style buttons from components
- Use light theme to add colour and theming properties
- Add component structure in components

**How to test**  
- Run storybook and see button get styled correctly, no error in theming (make sure theme is linked to local)
- Check new structure

![image](https://user-images.githubusercontent.com/1199101/105867936-ea738e80-5ff5-11eb-8d7c-04161ada2b5a.png)

![image](https://user-images.githubusercontent.com/1199101/105867833-ce6fed00-5ff5-11eb-89c6-ef2ff8e6cea5.png)
